pkg/scheduler/scheduler_test.go

```diff
@@ -1488,3 +1488,80 @@ func TestUpdatePod(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdatePodWithSecrets(t *testing.T) {
```

Comment:
Fine for now, but can you open a follow up PR to merge this with the test above into a single test table?

Reply:
I'll adjust it in this PR.